src/main/java/com/google/sps/servlets/BooksServlet.java

```diff
@@ -45,9 +46,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<String> terms = Arrays.asList(request.getParameterValues("key"));
     LinkedHashMap<String, List<Book>> booksMap = new LinkedHashMap<>();
     terms.forEach((term) -> {
-      String jsonString = getJsonStringForTerm(term);
-      List<Book> books = makeBooksList(jsonString);
-      booksMap.put(term, books);
+      try {
+        String jsonString = getJsonStringForTerm(term);
+        List<Book> books = makeBooksList(jsonString);
+        booksMap.put(term, books);
+      } catch(Exception e) {
```

Comment:
[Nit] As mentioned in previous PRs I'd really try to avoid using broad catch statement when possible.  Do we have a specific error we are worried about occurring here?

Reply:
Yeah, when there are no results there's a NullPointerException but when there's a problem with the API_KEY or with the API itself there's another one

Reply:
I see.  In that case,  I would recommend identifying the types of exceptions you expect and putting them in a multiple catch block
e.g.
`try {
  myFunctionThatThrowsNullPointerOrIoException()
}  catch (NullPointerException|IOException exception) {
  handleException(exception)
}`